lib/dream-date-formatter.js

```diff
@@ -2,6 +2,24 @@
 
 const ordinal = require('@quarterto/ordinal');
 
+const propertyNames = `
+	Y YY YYY YYYY
+	M MM MMM MMMM Mz Mo
+	D DD DDD DDDD Dz Do
+	d dd ddd dddd
+	H HH Hz h a
+	m mm mz
+	s ss sz
+	PD PT P
+	LT LTS
+	L LL LLL LLLL
+	l ll lll llll
+`.trim().split(/\s+/);
```

Comment:
Ignore the fact that this is a split string, it's a hangover from when I was tackling this differently. Should just be an array.